src/plugins/WebOfTrust/util/SynchronousDelayedBackgroundJob.java

```diff
@@ -0,0 +1,231 @@
+package plugins.WebOfTrust.util;
+
+/**
+ * Implementation of delayed background jobs where {@link #triggerExecution} blocks until the
+ * background job has finished, as if the background job was performed synchronously. This
+ * class is intended to be used in unit tests, and is likely not very useful for general purposes.
+ *
+ * @author bertm
+ */
+public class SynchronousDelayedBackgroundJob implements DelayedBackgroundJob {
```

Comment:
FYI, no merge blocker:
Advantages of making classes final whenever possible:
- Making a class final implicitely also makes all its functions final. Final functions have the advantage of not needing a <a href="http://en.wikipedia.org/wiki/Virtual_method_table">vtable</a>, i.e. you are speeding up function calls by that. It is a small optimization, but it is one.
- I also like the general concept of making everything write-protected which doesn't need to be written too, and if a class doesn't have to be extended yet, thats also falls in this category. 
- Also helps the developer to know that there is not more stuff to read.